src/main/java/internet/shop/controllers/order/AllUserOrdersController.java

```diff
@@ -0,0 +1,28 @@
+package internet.shop.controllers.order;
+
+import internet.shop.lib.Injector;
+import internet.shop.model.User;
+import internet.shop.service.OrderService;
+import internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AllUserOrdersController extends HttpServlet {
```

Comment:
```suggestion
public class GetUserOrdersController extends HttpServlet {
```
давай будемо притримуватись схожого неймінгу для котроллерів: CreateOrder, AddProduct etc.